modules/non_summary_pipeline.py

```diff
@@ -435,15 +435,19 @@ def from_json(file_name, element):
     """Returns an object from the JSON representation."""
 
     try:
-        return [(file_name, json.loads(element))]
+        return file_name, json.loads(element)
     except Exception as e:
         logging.error('Unable to parse file %s into JSON object "%s...": %s' % (file_name, element[:50], e))
-        return [(None)]
```

Comment:
I think the reason this might not have worked is that it's still a list of tuples, where the tuple contains None. I wonder if this would have worked `return [None]`

Reply:
Yeah I did wonder the same. But thought easier to revert the change and handle later. Though at a risk that the pipeline might fall down later. We'll see in a couple of hours!

Reply:
This is similar to the way [`FlatMap` worked](https://github.com/HTTPArchive/bigquery/blob/master/dataflow/python/bigquery_import.py#L358) for us in the previous pipeline. Simply [returning](https://github.com/HTTPArchive/bigquery/blob/master/dataflow/python/bigquery_import.py#L29) (None implicitly) seems to have worked.

Reply:
If the current on fails I'll try this. I'm not sure if it's broken JSONs, empty JSONs, or something else. But yeah should have a test with a few of those. Had a few things on this morning so figured easier to make the fix and kick it off while I wasn't able to look.

Reply:
Seems to be broken JSON. One hit in the logs for the error message above:

```
Unable to parse file gs://httparchive/crawls/android-Jun_1_2023/230614_Mx1RN_DN1RY.har.gz into JSON object "{"log":{"version":"1.1","creator":{"name":"WebPage...": Unterminated string starting at: line 1 column 1554282 (char 1554281)
```

First 50 chars in the log look normal so might be an encoding issue somewhere in there. Tried downloading ([CDN link](https://cdn.httparchive.org/crawls/android-Jun_1_2023/230614_Mx1RN_DN1RY.har.gz)) and unzipping but it's failing:

```
gunzip: 230614_Mx1RN_DN1RY.har.gz: unexpected end of file
gunzip: 230614_Mx1RN_DN1RY.har.gz: uncompress failed
```

Reply:
@pmeenan FYI you might want to look into how that HAR got corrupted in the first place

Reply:
How did you find that in the log? I can filter to find the error, but then could never find the text included in this filter to get the details (even when copying and pasting all the output to a text editor to make searching easier):

<img width="632" alt="image" src="https://github.com/HTTPArchive/data-pipeline/assets/10931297/8427b628-2dda-460f-93dd-e1d9f59779d0">

Reply:
In [Logs Explorer](https://console.cloud.google.com/logs/query;query=%22Unable%20to%20parse%20file%22;timeRange=P1D;cursorTimestamp=2023-06-28T10:56:26.244119405Z?project=httparchive) I searched `Unable to parse file` and set the time to last 1 day